pkg/k8sutil/operator.go

```diff
@@ -37,6 +39,16 @@ func RestartDeployment(ctx *Context, namespace, deploymentName string) {
 	fmt.Printf("deployment.apps/%s restarted\n", deploymentName)
 }
 
+func GetOperatorPodFromPodList(ctx *Context, operatorNamespace string) corev1.Pod {
+	opts := v1.ListOptions{LabelSelector: fmt.Sprintf("app=%s", "rook-ceph-operator")}
+	pods, err := ctx.Clientset.CoreV1().Pods(operatorNamespace).List(context.TODO(), opts)
+	if err != nil || len(pods.Items) == 0 || !pods.Items[0].DeletionTimestamp.IsZero() {
+		log.Error("failed to get rook operator pod where the command could be executed")
+		log.Fatal(err)
+	}
+	return pods.Items[0]
```

Comment:
How about iterating through the list of pods and finding the pod with status Running and not deleted? Otherwise, we have the problem again to get the terminating pod in that corner case.

Reply:
done